### PR TITLE
APP-4270 Block fill-rule="evenodd"  on svg files

### PR DESCRIPTION
**Pull request by symphony-annaLlorens**
### Description
- Remove current source files with still had the rule applied.
- Block the addition on new svg files with the prop: fill-rule="evenodd"
- Updated icon-font-generator lib

[Jira-ticket](https://perzoinc.atlassian.net/browse/APP-4270)

### CODE REVIEW

File: src/icons/svg/busy-small.svg [hunk 1]
```diff
@@ -1,3 +1,3 @@
...
+<path   d="M0.365731 1.41517C0.497293 0.890487 0.890487 0.497293 1.41517 0.365731C2.10035 0.193925 3.25929 0 5 0C6.74071 0 7.89965 0.193925 8.58483 0.365731C9.10951 0.497293 9.50271 0.890486 9.63427 1.41517C9.80608 2.10035 10 3.25929 10 5C10 6.74071 9.80608 7.89965 9.63427 8.58483C9.50271 9.10951 9.10951 9.50271 8.58483 9.63427C7.89965 9.80608 6.74071 10 5 10C3.25929 10 2.10035 9.80608 1.41517 9.63427C0.890487 9.50271 0.497293 9.10951 0.365731 8.58483C0.193925 7.89965 0 6.74071 0 5C0 3.25929 0.193925 2.10035 0.365731 1.41517ZM7.5 5.87501C7.98325 5.87501 8.375 5.48326 8.375 5.00001C8.375 4.51676 7.98325 4.12501 7.5 4.12501H2.5C2.01675 4.12501 1.625 4.51676 1.625 5.00001C1.625 5.48326 2.01675 5.87501 2.5 5.87501H7.5Z" fill="#EE3D3D"/>
```
Comment (symphony-adnane) [inline src/icons/svg/busy-small.svg:2]:
```suggestion
<path  d="M0.365731 1.41517C0.497293 0.890487 0.890487 0.497293 1.41517 0.365731C2.10035 0.193925 3.25929 0 5 0C6.74071 0 7.89965 0.193925 8.58483 0.365731C9.10951 0.497293 9.50271 0.890486 9.63427 1.41517C9.80608 2.10035 10 3.25929 10 5C10 6.74071 9.80608 7.89965 9.63427 8.58483C9.50271 9.10951 9.10951 9.50271 8.58483 9.63427C7.89965 9.80608 6.74071 10 5 10C3.25929 10 2.10035 9.80608 1.41517 9.63427C0.890487 9.50271 0.497293 9.10951 0.365731 8.58483C0.193925 7.89965 0 6.74071 0 5C0 3.25929 0.193925 2.10035 0.365731 1.41517ZM7.5 5.87501C7.98325 5.87501 8.375 5.48326 8.375 5.00001C8.375 4.51676 7.98325 4.12501 7.5 4.12501H2.5C2.01675 4.12501 1.625 4.51676 1.625 5.00001C1.625 5.48326 2.01675 5.87501 2.5 5.87501H7.5Z" fill="#EE3D3D"/>
```

File: src/scripts/generate-icons.js [hunk 2]
```diff
@@ -21,4 +21,20 @@ const generateIcons = () => {
   console.info('Auto-generated icons.stories.js ✅');
 };
 
+const blockSvgProp = (prop) => {
+  fs.readdirSync(`${SRC_ICONS}svg/`).forEach(file => {
+    fs.readFile(`${SRC_ICONS}svg/${file}`, 'utf8', (err, svgSrc) => {
+      try {
+        if(svgSrc.includes(prop)) {
+          throw `The file ${file} contains a not allowed css prop: ${prop}. Please, consider updating the source file`;
```
Comment (symphony-adnane) [inline src/scripts/generate-icons.js:29]: it's not really a CSS prop :)
We can talk about just about a forbidden property:
File xxx uses a forbidden property....

Comment (symphony-annaLlorens) [reply src/scripts/generate-icons.js:29]: Screenshot 2021-09-22 at 10 55 14

File: src/icons/svg/busy-small.svg (hunk 1 above)
Comment (symphony-jean-michael) [inline src/icons/svg/busy-small.svg:2]: Are you sure it will display correctly if you remove these properties? Do we need a new SVG file from the UX team?

Comment (symphony-adnane) [reply src/icons/svg/busy-small.svg:2]: it's already the new JM 
just that they kept the prop there but it's not used, 
Could you though double check that they still display properly @symphony-annaLlorens ?

**Review by symphony-adnane — APPROVED** [review]

**Review by symphony-jean-michael — APPROVED** [review]

**Pull request merged**